### PR TITLE
refactor(grafana): Adapt cloudalchemy role to TDP

### LINKED ISSUE #55: Develop a TOSIT role for grafana-server
External grafana role follows Ansible standard rules to notify service to reload when configuration is modified. 

In TDP we do not follow these rules and use tdp-lib to launch specific operation in order.
We should not use external role to install grafana in order to have a good integration in TDP.

### PR DESCRIPTION
**Pull request by giem-git**
#### Which issue(s) this PR fixes

Fixes #55

#### Agreements

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: requirements.yml
Comment (kpgtek) [inline requirements.yml:1]: If this file is removed, I think there wil need to modify the [setup.sh](https://github.com/TOSIT-IO/tdp-observability/blob/main/scripts/setup.sh) script (installation by ```ansible-galaxy```).

**Review by kpgtek — CHANGES_REQUESTED** [review]

File: requirements.yml
Comment (giem-git) [reply requirements.yml:1]: setup has been modified

File: roles/grafana/server/tasks/plugins.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+- name: Check which plugins are installed
+  ansible.builtin.find:
+    file_type: directory
+    recurse: false
+    paths: "{{ grafana_data_dir }}/plugins"
+  register: installed_plugins
+
+- name: Install plugins
+  become: true
```
[the remark this answers is not in the document]
Comment (giem-git) [reply roles/grafana/server/tasks/plugins.yml:14]: this is the file from cloudalchemy role. We dont use plugins in tdp, so we won't fix this

File: roles/grafana/server/templates/tmpfiles.j2 [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{#
+# Copyright 2022 TOSIT.IO
+# Copyright 2017-2018 Pawel Krupa, Roman Demachkovych
+# SPDX-License-Identifier: Apache-2.0 AND MIT
+# Copied and adapted from https://github.com/cloudalchemy/ansible-grafana.git
+#}
+{{ ansible_managed | comment }}
+d {{ grafana_server.socket | dirname }} 0775 grafana grafana
```
[the remark this answers is not in the document]
Comment (giem-git) [reply roles/grafana/server/templates/tmpfiles.j2]: fixed

**Review by rpignolet — APPROVED** [review]

**Review by kpgtek — APPROVED** [review]

**Pull request merged**